examples/tools/bfield_type.hpp

```diff
@@ -0,0 +1,21 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2025 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#include <covfie/core/algebra/affine.hpp>
+#include <covfie/core/backend/primitive/array.hpp>
+#include <covfie/core/backend/transformer/affine.hpp>
+#include <covfie/core/backend/transformer/nearest_neighbour.hpp>
+#include <covfie/core/backend/transformer/strided.hpp>
+#include <covfie/core/field.hpp>
+#include <covfie/core/parameter_pack.hpp>
+
+// TODO: Put these types in traccc core.
+using backend_t = covfie::backend::affine<covfie::backend::nearest_neighbour<
```

Comment:
Can we use the linear approximation instead of NN? We can include NN after we confirm that physics performance stays good

Reply:
We can use the nearest-neighbour to make building the b-field easier, but we can then convert it to linear interpolation afterwards. But you are right the type that lives in the core should not be this NN type.

Reply:
I removed the comment saying this should go into the core lib; we can use a linearly interpolated type in core.